src/kubernetes-ingress-controller/guides/using-gateway-api.md

```diff
@@ -103,16 +103,31 @@ spec:
   listeners:
   - name: proxy
     port: 80
+	hostname: kong.example
```

Comment:
If I see this right, the indentation here is using tabs instead of spaces (as in other lines of this snippet). I believe we should be consistent and use only one way of expressing whitespace.

```suggestion
    hostname: kong.example
```